### DISCUSSION TITLE
Want to resize the player

### DISCUSSION THREAD

**Discussion by restareaByWeezy** [gilbarbara/react-spotify-web-playback#106] [Q&A] [answered]
I tried to change the player's width by wrapping playercomponent.
But figured out that player's width depends on the length of song title.
I want to fix the size for my player. 
Is there any solution?
(also I want to know what loaderSize do)
example
![image](https://user-images.githubusercontent.com/64529155/174001504-19790116-4f9a-412f-82df-e8ee35253841.png)
![image](https://user-images.githubusercontent.com/64529155/174001544-896d7d65-0c2c-47cf-adc1-e849e383870c.png)

```js
 <div className={styles.playerWrapper}>
          <SpotifyWebPlayer
            initialVolume={0.5}
            token={token}
            uris={playUri}
            styles={{
              activeColor: '#fff',
              bgColor: '#333',
              color: '#fff',
              loaderColor: '#fff',
              sliderColor: '#1cb954',
              trackArtistColor: '#ccc',
              trackNameColor: '#fff',
            }}
          />
        </div>
```

**Comment by gilbarbara** [comment, marked as answer]
Hey @restareaByWeezy 
I've implemented the "compact" layout in version 0.12 if you still need it.
https://react-spotify-web-playback.gilbarbara.dev/
